src/rpc/blockchain.cpp

```diff
@@ -352,7 +342,7 @@ static UniValue getdifficulty(const JSONRPCRequest& request)
         );
 
     LOCK(cs_main);
-    return GetDifficulty();
+    return GetDifficulty(chainActive.Tip());
```

Comment:
style nit: Personally I use `::chainActive`

Reply:
I like this too but I'm going to leave this PR be. I have a follow-up I think will accommodate it better.